packages/instantsearch.js/src/components/Hits/Hits.tsx

```diff
@@ -53,7 +53,12 @@ const Hits = ({
             {...templateProps}
             templateKey="item"
             rootTagName="li"
-            rootProps={{ className: cssClasses.item }}
+            rootProps={{
+              className: cssClasses.item,
+              onClick: () => {
+                sendEvent('click', hit, 'Hit Clicked');
```

Comment:
```suggestion
                sendEvent('click', [hit], 'Hit Clicked');
```

are these two equivalent?

Reply:
The `sendEvent` accepts both a single hit or an array of them, so I guess just passing the hit is fine?